tkg/clusterclient/awsIngressRules.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package clusterclient
+
+import (
+	capav1beta2 "sigs.k8s.io/cluster-api-provider-aws/v2/api/v1beta2"
+)
+
+const (
+	DefaultKappControllerHostPort = 10100
+	DefaultAddonsManagerHostPort  = 9865
+	DefaultFeatureGateHostPort    = 9443
+)
+
+var (
+	awsIngressRules = []capav1beta2.CNIIngressRule{
+		capav1beta2.CNIIngressRule{
+			Description: "kapp-controller",
+			Protocol:    capav1beta2.SecurityGroupProtocolTCP,
+			FromPort:    DefaultKappControllerHostPort,
+			ToPort:      DefaultKappControllerHostPort,
+		},
+		capav1beta2.CNIIngressRule{
+			Description: "addons-manager",
+			Protocol:    capav1beta2.SecurityGroupProtocolTCP,
+			FromPort:    DefaultAddonsManagerHostPort,
+			ToPort:      DefaultAddonsManagerHostPort,
+		},
+		capav1beta2.CNIIngressRule{
```

Comment:
I am not sure if feature gates need it since they were planning to remove host network. @yharish991 @danniel1205 Could you please confirm ?

Reply:
yes, removing the host network is a potential solution we are exploring for featuregates, I think we can remove the featuregates ingress rule here in this pr and can add it later if needed.

Reply:
I will leave it commented out until we get this pr ready to merge.

Reply:
@adduarte we merged the potential solution that i was talking about for featuregates today with this pr https://github.com/vmware-tanzu/tanzu-framework/pull/4039, so we can skip adding the featuregates webhook server port here in this pr..thanks :)

Reply:
will remove feature-gate

Reply:
done